src/averageAges.js

```diff
@@ -20,6 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const searchedMens = century
+    ? people.filter(person => person.sex === 'm'
+    && century === Math.ceil(person.died / 100))
+    : people.filter(person => person.sex === 'm');
```

Comment:
2. [CODE STYLE] - If you move your shortened return statement in arrow function to a newline, use round brackets

BAD EXAMPLE:
```
const humans = people.map((a, b) =>
    a + b);

const babies = people.map(person => children 
 ? a
 : b
 
```

GOOD EXAMPLE: 
```
const humans = people.reduce((a, b) => (
  a + b
), 0);